src/test/java/org/breedinginsight/model/delta/DeltaEntityFactoryUnitTest.java

```diff
@@ -0,0 +1,461 @@
+                .license("license")
+                .locationDbId("4abea286-a93a-44ca-812f-5420106a71c3")
+                .locationName("location")
+                .observationLevels(List.of(createBrAPIObservationUnitHierarchyLevel()))
+                .observationUnitsDescription("units description")
+                .observationVariableDbIds(List.of("67f6449b-b79f-4126-91bf-1b3903571a3f", "0684c99e-a01a-4f59-8c14-bbe3e550c8c5"))
+                .seasons(List.of("2021Fall", "2022Fall"))
+                .startDate(OffsetDateTime.now())
+                .studyCode("study code")
+                .studyDescription("description")
+                .studyName("Study name")
+                .studyPUI("d5295e32-da72-433f-b3bd-a40cf5b93bbb")
+                .studyType("phenotyping trial")
+                .trialDbId("703cb8d8-9167-457c-8934-6d065415d312")
+                .trialName("Snail Trial");
+    }
+
+    private BrAPITrial createBrAPITrial() {
+        String additionalInfoString = "{\"datasets\":[{\"id\":\"0d9f03bf-4b0c-40e8-95b9-9ca0a107f30f\",\"name\":\"Plot\",\"level\":\"0\"}],\"createdBy\":{\"userId\":\"e3f92938-fae4-4d57-93c6-11970a8128a6\",\"userName\":\"BI-DEV Admin\"},\"createdDate\":\"2024-07-12\",\"experimentType\":\"Disease resistance screening\",\"experimentNumber\":\"1\",\"defaultObservationLevel\":\"Plot\"}";
+
+        return new BrAPITrial()
+                .trialDbId("429ff1cd-1a4a-4be7-bd6c-d10047d03230")
+                .active(true)
+                .additionalInfo(toJsonObject(additionalInfoString))
+                .commonCropName("Snail")
+                .contacts(List.of(createBrAPIContact()))
+                .datasetAuthorships(List.of(createBrAPITrialDatasetAuthorship()))
+                .documentationURL("http://localhost")
+                .endDate(LocalDate.now())
+                .externalReferences(createExternalReferences())
+                .programDbId("401ab961-4c37-47f0-910b-0ea39648f547")
+                .programName("Large Program")
+                .publications(List.of(createBrAPITrialPublication()))
+                .startDate(LocalDate.now())
+                .trialDescription("the description")
+                .trialName("Snail Trial")
+                .trialPUI("9bfa3dba-5195-4a08-a891-cd2b105116fa");
+
+    }
+
+    private BrAPIObservation createBrAPIObservation() {
+        String additionalInfoString = "{\"createdBy\":{\"userId\":\"e3f92938-fae4-4d57-93c6-11970a8128a6\",\"userName\":\"BI-DEV Admin\"},\"studyName\":\"Salinas, CA 2022\",\"createdDate\":\"2024-07-12T09:44:23.34292-04:00\"}";
+
+        return new BrAPIObservation()
+            .observationDbId("192161b2-3f89-499b-8f04-7298128e9f1a")
+            .additionalInfo(toJsonObject(additionalInfoString))
+            .collector("intern")
+            .externalReferences(createExternalReferences())
+            .geoCoordinates(BrApiGeoJSON.builder().geometry(null).type("Feature").build())
```

Comment:
This is showing as a compile error because geoCoordinates has private access.

Reply:
I added getter, setter and builder methods for `BrAPIObservation` in the [brapi PR that was merged](https://github.com/Breeding-Insight/brapi/pull/224/files), you may need to run `clean install -U` (see https://maven.apache.org/ref/3.9.8/maven-embedder/cli.html) or even purge your local maven cache (usually ~/.m2) if the new snapshot still isn't being picked up.